Answers.md

```diff
@@ -0,0 +1,9 @@
+If I were to describe semantic HTML, I would say that semantic HTML is a way of creating div's that have meaning. For example, a <p> is a stands for paragraph. It also allows code to be more readable by users and allows for better SEO.
+
```

Comment:
You explained it correctly, but gave the wrong example haha. 

An example would be more like 
`<article>`
`<address>`
`<section>`
Things like those